### ISSUE TITLE
Consolidate apiserver readiness checks

### ISSUE THREAD

**Issue by davidewatson** [samsung-cnct/kraken-lib#229]
We currently wait for the API server to be read in 3 different places:

```
rodin:k2 dwat$ grep -r 'Wait for api server' ansible
ansible/roles/kraken.fabric/kraken.fabric.flannel/tasks/main.yml:- name: Wait for api server to become available in case it's not
ansible/roles/kraken.readiness/tasks/do-wait.yaml:- name: Wait for api server to become available in case it's not
ansible/roles/kraken.services/tasks/run-services.yaml:- name: Wait for api server to become available in case it's not
rodin:k2 dwat$ 
```

One of these has diverged from the rest which suggests that it may be wrong. We should remove duplications like these from the code to make it is easier to maintain.

**Comment by coffeepac** [comment]
wait for etcd/control plane to be stabilized (0.3)

**Comment by davidewatson** [comment]
May be fixed by #540.
